scripts/criu-ns

```diff
@@ -122,9 +123,39 @@ def wrap_restore():
     if '--restore-detached' in restore_args:
         restore_detached = True
         restore_args.remove('--restore-detached')
+    restore_pidfile = None
+    if '--pidfile' in restore_args:
+        try:
+            opt_index = restore_args.index('--pidfile')
+            restore_pidfile = restore_args[opt_index + 1]
+            del restore_args[opt_index:opt_index + 2]
+        except (ValueError, IndexError, FileNotFoundError):
+            raise OSError(errno.ENOENT, "--pidfile missing argument")
+        if restore_pidfile[0] != '/':
+            base_dir_opt = None
+            if '--work-dir' in restore_args:
+                base_dir_opt = '--work-dir'
+            elif '-W' in restore_args:
+                base_dir_opt = '-W'
+            elif '--images-dir' in restore_args:
+                base_dir_opt = '--images-dir'
+            elif '-D' in restore_args:
+                base_dir_opt = '-D'
```

Comment:
this could be simplified to something like:
```Python
if not restore_pidfile.startswith('/'):
    for base_dir_opt in ['--work-dir', '-W', '--images-dir', '-D']:
        if base_dir_opt in restore_args:
            try:
                opt_index = restore_args.index(base_dir_opt)
                restore_pidfile = os.path.join(restore_args[opt_index + 1], restore_pidfile)
                break
            except (ValueError, IndexError, FileNotFoundError):
                raise OSError(errno.ENOENT, base_dir_opt + " missing argument")
```